libs/wingc/src/lsp/completions.rs

```diff
@@ -330,13 +331,14 @@ fn get_completions_from_namespace(
 		LookupResult::DefinedLater => vec![],
 		LookupResult::ExpectedNamespace(_) => vec![],
 	};
-	namespace
+	let l: Vec<CompletionItem> = namespace
 		.env
 		.symbol_map
 		.iter()
 		.flat_map(|(name, symbol)| format_symbol_kind_as_completion(name, &symbol.1))
 		.chain(util_completions.into_iter())
-		.collect()
+		.collect();
+	l
```

Comment:
nit: A most rusty way to do this might be to call `collect_vec` (requires importing a trait from `itertools`) or specifying the collection type directly with `collect::<Vec<_>>()` so that the temporary variable isn't needed

Reply:
ah yeah this was better before, I added the temp var for debugging. If I take it out and just return the collect it's cleaner